editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5842,7 +5842,7 @@ void CanvasItemEditorViewport::_perform_drop_data() {
 	Vector<String> error_files;
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-	undo_redo->create_action(TTR("Create Node"));
+	undo_redo->create_action_for_history(TTR("Create Node"), EditorNode::get_singleton()->get_editor_data().get_current_edited_scene_history_id());
```

Comment:
```suggestion
	undo_redo->create_action_for_history(TTR("Create Node"), EditorNode::get_editor_data().get_current_edited_scene_history_id());
```
This method is static.

Reply:
Fixed 👍